src/pages/gutenberg/index.vue

```diff
@@ -367,11 +367,16 @@
       </div>
     </section>
     <div :class="['flex', 'justify-center', 'mt-[220px]']">
-      <GutenbergButton
-        :text="$t('gutenberg_download_more')"
-        preset="primary"
-        @click="handleClickMore"
-      />
+      <NuxtLink
+        :to="localeLocation({ name: 'gutenberg-free-audio-books' })"
+        target="_blank"
```

Comment:
Original behavior is to open in same window?

Reply:
[69e3201](https://github.com/likecoin/liker-land/pull/1491/commits/69e32012d5cd980cbc055e8eebe9d0f8befcc789)